includes/class-sensei-lesson.php

```diff
@@ -4266,7 +4266,7 @@ public static function footer_quiz_call_to_action( $lesson_id = 0, $user_id = 0
 
 					<p>
 
-						<a class="button"
+						<a class="button quiz-view"
```

Comment:
Could we call this class `sensei-lesson__view-quiz` instead?

Reply:
Sounds good! Thank you.